bitnami/mariadb-galera/templates/statefulset.yaml

```
@@ -242,6 +242,8 @@ spec:
           image: {{ template "mariadb-galera.metrics.image" . }}
           imagePullPolicy: {{ .Values.metrics.image.pullPolicy | quote }}
           env:
+            - name: MARIADB_METRICS_EXTRA_FLAGS
+              value: {{ default "" .Values.metrics.extraFlags | quote }}
```

If you treat `.Values.metrics.extraFlags` as an array, use:
```suggestion
            - name: MARIADB_METRICS_EXTRA_FLAGS
              value: {{ default "" (join " " .Values.metrics.extraFlags) | quote }}
```